bflibrary/src/x86-win-sdl2/sscreen.c

```diff
@@ -480,6 +480,10 @@ TbResult LbScreenSetupAnyMode(TbScreenMode mode, TbScreenCoord width,
         if (new_fullscreen_flags == 0)
         {
             SDL_SetWindowSize(lbWindow, mdWidth, mdHeight);
+            if ((SDL_getenv("SDL_VIDEO_CENTERED")) && (strcmp((SDL_getenv("SDL_VIDEO_CENTERED")), "1") == 0))
```

Comment:
Would be better to read the variable earlier, before `if (lbWindow != NULL)`. After all we should use it regardless of whether the window already existed or not.

Also it is best to call getenv once - store it in a variable. And looking at the SDL2 code, I see there's a function which will directly give us a boolean rather than a string:
```
TbBool wnd_centered;
[...]
wnd_centered = SDL_GetHintBoolean("SDL_VIDEO_CENTERED", SDL_FALSE);
[...]
if (wnd_centered)
```
EDIT:
proposed even better solution below.